app/Http/Controllers/api/RecruiterController.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Exceptions\UserNotAuthenticatedException;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UpdateRecruiterRequest;
+use App\Http\Requests\UserRequest;
+use App\Http\Resources\RecruiterListResource;
+use App\Http\Resources\RecruiterResource;
+use App\Models\Recruiter;
+use App\Models\User;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+
+class RecruiterController extends Controller
+{
+    public function index()
+    {
+        try {
+            $recruitersList = Recruiter::all();
+
+            if ($recruitersList->isEmpty()) {
+
+                throw new ModelNotFoundException(__('No hi ha reclutadors a la base de dades.'), 404);
+
+            }
+
+            return response()->json([
+                'data' => RecruiterListResource::collection($recruitersList)], 200);
+
+        } catch(ModelNotFoundException $modelNotFoundException) {
+            return response()->json(['message' => $modelNotFoundException->getMessage()], $modelNotFoundException->getCode());
+        }
+
+    }
+
+    public function store(UserRequest $request)
+    {
+
+        $recruiter = DB::transaction(function () use ($request) {
+
+            $user = User::create([
+                'name' => Str::lower($request->name),
+                'surname' => Str::lower($request->surname),
+                'dni' => Str::lower($request->dni),
+                'email' => Str::lower($request->email),
+                'password' => bcrypt($request->password),
+                'active' => true,
+            ]);
+
+            $recruiter = $user->recruiter()->create([
+                'company' => $request->company,
+                'sector' => $request->sector,
+            ]);
+
+            $user->assignRole('recruiter');
+
+            return $recruiter;
+
+        });
+
+        if (!$recruiter) {
+            throw new HttpResponseException(response()->json([
+                'message' => __(
+                    'Registre no efectuat. Si us plau, torna-ho a provar.'
+                )], 404));
+        }
+
+        return response()->json([
+            'message' => __(
+                'Registre realitzat amb èxit.'
+            )], 201);
+    }
+
+    public function show($id)
+    {
+        try {
+            $recruiter = Recruiter::where('id', $id)->first();
+
+            if (!$recruiter) {
+                throw new ModelNotFoundException(__('Usuari no trobat.'), 404);
+            }
+
+            return response()->json([
+                'data' => RecruiterResource::make($recruiter)], 200);
```

Comment:
I think i'd change this whole logic. Instead of transforming name to lowercase and then change the first letter to uppercase by using your brand new RecruiterResource.php and RecruiterListResource.php I'd keep the fields as users would like to do. Imagine a person who es called "John O'Conner" or "Juan Carlos de Dios". Let's avoid headaches and don't add those files 😉